table-to-blt.mjs

```diff
@@ -37,15 +37,15 @@
 
 import { JSDOM } from 'jsdom';
 
-async function main(file, nbseats, electionname, sortballots = true, shownames = false) {
+export async function main(file, nbseats, electionname, sortballots = true, shownames = false) {
     if (isNaN(nbseats)) {
        throw new Error(`Required number of seats missing.`);
     }
     const dom = await JSDOM.fromFile(file);
-    generateBLT(dom.window.document, nbseats, electionname, sortballots, shownames);
+    return generateBLT(dom.window.document, nbseats, electionname, sortballots, shownames);
```

Comment:
instead of having `generateBLT` outputting directly on the console, we have it return the lines that it is generating and a warnings as a second return value. This makes it easier to test (as generally does keeping side-effects as local as possible)